src/WorkItems/ServiceProviderFactory.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
```

Comment:
>Extensions [](start = 16, length = 10)

System directives come first. (Change your VS settings to make this a default). Add copyright header to this file.

Reply:
Added copyright and sorted using directives for the files.

---
In reply to: [396823390](https://github.com/microsoft/sarif-sdk/pull/1820#discussion_r396823390) [](ancestors = 396823390)